tools/serverpod_cli/lib/src/test_util/builders/generator_config_builder.dart

```diff
@@ -103,6 +105,12 @@ class GeneratorConfigBuilder {
     return this;
   }
 
+  GeneratorConfigBuilder withRelativeServerTestToolsPathParts(
+      List<String> relativeServerTestToolsPathParts) {
```

Comment:
Suggestion(NIT): Add comma after last parameter.

```suggestion
  GeneratorConfigBuilder withRelativeServerTestToolsPathParts(
      List<String> relativeServerTestToolsPathParts,) {
```